### PR TITLE
feat: Svg handler

### PR DESCRIPTION
**Pull request by Victorportus**
SVG handler incorporated.

We upload the SVG on the assets/icons folder, save them with a Capital letter at the beginning. Latter import the icon where you want to use it and add it. 

I added 2 Icons, waiting for designers to give the rest of them. 

Screenshot 2024-06-25 at 4 39 31 PM

And updated the arrow on the notifications page.

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b0z9hrm SVG on React Native](https://app.clickup.com/t/86b0z9hrm)

File: android/gradle.properties [hunk 1]
```diff
@@ -55,7 +55,4 @@ EX_DEV_CLIENT_NETWORK_INSPECTOR=true
 # Use legacy packaging to compress native libraries in the resulting APK.
 expo.useLegacyPackaging=false
 
-# Comment out for Mac
-org.gradle.jvmargs=-Djava.io.tmpdir=C:/temp/gradle -Dorg.sqlite.tmpdir=C:/temp/gradle
-
```
Comment (whamester) [inline android/gradle.properties]: I think Rein and Beck need this config here. for us we have to comment out that line when we want to run the app locally

File: src/app/(tabs)/_layout.tsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import { Tabs } from "expo-router";
+import Home_icon from "assets/icons/Home_icon.svg";
```
Comment (whamester) [inline src/app/(tabs)/_layout.tsx]: let's rename these to follow the component naming convention

HomeIcon ... HomeIcon.svg

ArrowLeftIcon ... ArrowLeftIcon.svg

**Review by whamester — CHANGES_REQUESTED** [review]
I see that there are some changes inside the ios folder, when the comments are addressed I'll test this branch and regenerate the ios folder to make sure it's working with the rest of the features.

Thanks great job!

File: src/app/(tabs)/_layout.tsx (hunk 2 above)
Comment (Victorportus) [reply src/app/(tabs)/_layout.tsx]: Those where only examples, I already got the SVG's from Designer and we will implement all ths SVG according to that name convention

File: android/gradle.properties (hunk 1 above)
Comment (Victorportus) [reply android/gradle.properties]: I added the line back, but I have it commented. If I have it uncommented, it does not run on my device

File: src/app/(tabs)/_layout.tsx (hunk 2 above)
Comment (Victorportus) [reply src/app/(tabs)/_layout.tsx]: I did a new commit with all the SVG and the name according to the conventions

**Review by whamester — APPROVED** [review]

**Review by ReinhardtBotha — APPROVED** [review]
Looks great! ✅

**Review by zeckkimon — APPROVED** [review]
Awesome!

**Pull request merged**